docs/documentation/development/bindings/xml-reference.md

```diff
@@ -101,6 +101,19 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>criteria.name</td><td>The name of the context related filter.</td></tr>  
 </table>
 
+### Supported contexts
+
+Context is used to provide some semantic details about the parameter. The UI use it to render different kind of input widgets. Currently, PaperUI supports following contexts:
+
+<table><tr><th>Name</th><th>Type</th><th>Format</th></tr>
+  <tr><td>Date</td><td>Text</td><td>DD:MM:YYYY</td></tr>
+  <tr><td>Time</td><td>Text/Integer</td><td>hh:mm</td></tr>
+  <tr><td>Color</td><td>Text</td><td>#000000 - #ffffff (hex color)</td></tr>
+  <tr><td>Item</td><td>Text</td><td>Item name</td></tr>
+  <tr><td>Thing</td><td>Text</td><td>UID of thing</td></tr>
+  <tr><td>Day of week</td><td>Text</td><td>MON, TUE, WED, THU, FRI, SAT, SUN <br></td></tr>
```

Comment:
Does the Paper UI handle this correctly? As you previously said that for German locale different Strings would apply here. If set to German locale, did you ensure that the Paper UI is still setting these English TLAs?